packages/resource/README.md

```diff
@@ -65,6 +66,24 @@ const pages = createAggregated(currentPage, []);
 
 Objects and Arrays are re-created on each operation, but the values will be left untouched, so `<For>` should work fine.
 
+### createDeepSignal
+
+This is an optimized implementation of what is required by the [resource storage option](https://www.solidjs.com/docs/latest/api#createresource:~:text=Resources%20can%20be%20set%20with%20custom%20defined%20storage):
```

Comment:
This doesn't really explain much of what `createDeepSignal` actually does. Besides the fact that it's meant to be used with the `storage` option.
The `storage` option lets you use something else besides a simple signal for storing the response, it doesn't assume anything about what it should implement besides the signal interface.
So it's not really an "optimized implementation", because there is nothing to compare against.
I think it should instead say that it's creating a store and will reconcile new responses so that the changes are fine-grain, and references will be preserved so using `<For>` is possible.

Reply:
Actually, it is size-optimized compared to the implementation in the solid docs (185bytes vs. 212bytes). But that point is probably moot.